rpc_jobs/rpc_aio.yml

```diff
@@ -193,10 +193,11 @@
            {TRIGGER_USER_VARS}
          UPGRADE_FROM_REF: "{UPGRADE_FROM_REF}"
       - rpc_gating_params
-      - single_use_slave_params:
+      - instance_params:
          IMAGE: "{IMAGE}"
          FLAVOR: "{FLAVOR}"
          REGION: "{REGION}"
+      - jenkins_node_params
```

Comment:
I was having trouble combining default and bool filters in [ansible](https://github.com/rcbops/rpc-gating/pull/228/files#diff-5be9faccf8f48d89f54f9248ca4de6e3R5), so I decided to default the values in all jobs so that I didn't have to worry about it in ansible. However this does add one more thing that all new jobs must do, so I'll look again at finding a way that doesn't rely on those variables being set.